deno/rest/v10/monetization.ts

```diff
@@ -0,0 +1,91 @@
+import type { Snowflake } from '../../globals.ts';
+import type { APIEntitlement, APISKU } from '../../v10.ts';
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#list-entitlements
+ */
+export interface RESTListAPIEntitlementsQuery {
+	/**
+	 * User ID to look up entitlements for
+	 */
+	user_id?: Snowflake | undefined;
+	/**
+	 * Optional list of SKU IDs to check entitlements for
+	 */
+	sku_ids?: Snowflake[] | undefined;
+	/**
+	 * Retrieve entitlements before this time
+	 */
+	before?: Snowflake | undefined;
+	/**
+	 * Retrieve entitlements after this time
+	 */
+	after?: Snowflake | undefined;
+	/**
+	 * Number of entitlements to return, 1-100, default 100
+	 */
+	limit?: number | undefined;
+	/**
+	 * Guild ID to look up entitlements for
+	 */
+	guild_id?: Snowflake | undefined;
+	/**
+	 * Whether entitlements should be omitted
```

Comment:
this doesn't seem right to me

Reply:
Copied from [api docs](https://discord.com/developers/docs/monetization/entitlements#list-entitlements)